app/src/main/assets/solarized.css

```diff
@@ -249,7 +257,7 @@ header.mbm {
   white-space: pre-wrap;
 }
 
-#article blockquote *, #article pre * {
+#article pre *  {
```

Comment:
Can you please remove the extra space, it's driving me nuts :-D

Reply:
Oops, sorry, see comment re. rushing. Done.